exercises/hello-world/description.md

```diff
@@ -1,46 +1,11 @@
 ["Hello, World!"](http://en.wikipedia.org/wiki/%22Hello,_world!%22_program) is
-the traditional first program for beginning programming in a new language.
+the traditional first program for beginning programming in a new language
+or environment.
 
-**Note:** You can skip this exercise by running:
+The objectives are simple:
 
-    exercism skip $TRACK_ID hello-world
+- Write a function that says "Hello, World!".
+- Run the tests and make sure that they succeed.
```

Comment:
Maybe "Run the tests" should be "Run the test(s)", as I expect most languages to only have a single test.